### ISSUE TITLE
wk_3 feb_epic BUG: MTL DEMO Model Parameters File Needs SP Floors Data

### ISSUE THREAD

**Issue by jamesmrollins** [lzim/teampsd#1698]
**1. MTL DEMO ModelParameters.xlsx Needs SP Floors Data:**  Just need someone to tell me which Model Parameters file to download from BISL, and I can take care of the rest.  I am not sure if you used a particular test case that you are familiar with, in terms of supporting any presentations. See attached for current file.

[ModelParameters (1).xlsx](https://github.com/lzim/teampsd/files/5963540/ModelParameters.1.xlsx)
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
iix. **Releases** - assign to key team release grouped with other relevant issues  

**3. If you anticipate that your **issue card** or **bug card** will have **dependencies** these can be added once the issue has been submitted.   
ix. **Add dependencies** - assess for and add dependencies *blocking* & *blocked by* this issue

Blocking 
#1637 
#1382 
#1591 
#1451

**Comment by lzim** [comment]
@jamesmrollins 
@staceypark 

Shouldn’t the best test file be the one here?
What’s difference between the two files: “abc” and  “ModelParameters.xlsx” ?

At present, I can only be sure of SP Parameters referrals flows for 640ao, in terms of field knowledge. But, I we should start testing MTL Red teams ASAP.


https://github.com/lzim/mtl/tree/master/blue/mtl2.0_models

**Comment by jamesmrollins** [comment]
Hi @staceypark and @lzim .  That MP file works for me!  There is no difference between the files.  It is in the directory because several of our auto-routines require a file named "ModelParameters.xlsx" otherwise, Vensim crashes.

**Comment by lzim** [comment]
@staceypark 

Is there anything else we need to discuss for this?

**Comment by jamesmrollins** [comment]
@lzim - the guidance here is not clear to me.  

The link in "here" I assume, is the blue directory shown at the bottom message.  That would be this file 100a1_abc_teama_All_2020_03_10.xlsx, correct?

I have checked available 640ao model parameters files in PROD, and they don't contain SP counts.  I don't think I understand your direction to test the file.

I was waiting for Stacey's input to bring clarification . . . 

Can I use the 100a1 file noted above?

Thanks,
James

**Comment by lzim** [comment]
I see @jamesmrollins 

I just pinged @staceypark on another issue to determine whether she needs my help to create this data if it is not available.

But, I can't seem to find where I just made that comment.

**Comment by jamesmrollins** [comment]
Hi @lzim , @lijenn , @staceypark , @branscombj , @dlounsbu 

The table below describes all the places that 100a1_abc_teama_All_2020_03_10.xlsx is currently being used.  I intend to change the name of this file to incorporate all lower case letters, IAW our team standards. 

filename | worldname | groupname
-- | -- | --
100a1_abc_teama_All_2020_03_10.xlsx | World001 | david_ind
100a1_abc_teama_All_2020_03_10.xlsx | World002 | jane_ind
100a1_abc_teama_All_2020_03_10.xlsx | World001 | dev_test_team
100a1_abc_teama_All_2020_03_10.xlsx | World008 | individual_login
100a1_abc_teama_All_2020_03_10.xlsx | World001 | mtl_facilitate_2020_09_ind
